src/tests/global_variables/env_variables_defaults.py

```diff
@@ -18,7 +18,7 @@ class _EnvVariables(DataPool, ABC):
     pull_secret: EnvVar = EnvVar(["PULL_SECRET"])
     deploy_target: EnvVar = EnvVar(["DEPLOY_TARGET"])
     offline_token: EnvVar = EnvVar(["OFFLINE_TOKEN"])
-    discovery_kernel_arguments: EnvVar = EnvVar(["DISCOVERY_KERNEL_ARGUMENTS"], loader=json.loads)
+    kernel_arguments: EnvVar = EnvVar(["DISCOVERY_KERNEL_ARGUMENTS"], loader=json.loads)
```

Comment:
```suggestion
    kernel_arguments: EnvVar = EnvVar(["KERNEL_ARGUMENTS"], loader=json.loads)
```
Change also on `skipper.env`

Reply:
done thanks :)